pkg/manager/member/startscript/v1/render_script.go

```diff
@@ -233,9 +233,36 @@ done
 }
 
 func RenderTiFlashStartScript(tc *v1alpha1.TidbCluster) (string, error) {
+	if tc.Spec.TiFlash.IsInitScriptDisabled() {
+		return RenderTiFlashStartScriptWithStartArgs(tc)
```

Comment:
Does this work well for `tc.AcrossK8s()` / `tc.IsTLSClusterEnabled()` as we special handle in `RenderTiFlashInitScript`

Reply:
fixed https://github.com/pingcap/tidb-operator/pull/5552/commits/5516f8fb723dfa40d9ef1912a963e562cbb7c945